docs/docs/components/radio.mdx

```diff
@@ -0,0 +1,16 @@
+# Radio (spec)
+
+# Notes
+
+- Single radio component.
+- Will contain Label.
+- Will have to provide some type of prop to configure the color of the selected radio button.
+
+| Attributes  |  Values  | Optional ? |
+| :---------- | :------: | ---------: |
+| `id`        |  string  |        Yes |
+| `name`      |  string  |         No |
+| `value`     |  string  |         No |
+| `label`     |  string  |         No |
```

Comment:
label should be provided as a separate component as `<Label>` it helps keep label and input far away and customisable layout wise

Reply:
Yeah ok that makes sense. Will change it.